### PR TITLE
feat: add slack channel configuration

### PR DESCRIPTION
**Pull request by JustasVersekys**
Added aws-chatbot integration.
For finishing this task to fully automated [issue](https://github.com/aws/aws-cdk/issues/7983) must be resolved.

Added three paramaters:
1. aws ssm put-parameter --name sphinx-env-master --value "C0138PMAGJZ" --type SecureString
2. aws ssm put-parameter --name sphinx-env-bumastemra-uat --value "C0131GJ9WET" --type SecureString
3. aws ssm put-parameter --name sphinx-env-other --value "C013NEH9YKT" --type SecureString

### CODE REVIEW

File: src/PipeLineTemplate/lib/notification.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        }
+
+        let slackChannelId = ssm.StringParameter.valueForSecureStringParameter(scope, slackChannelName, 1);
+
+        new chatbot.CfnSlackChannelConfiguration(this, props.projectName + 'SlackChannelConfiguration', {
+            loggingLevel: 'INFO',
+            configurationName: props.projectName + 'CodePipelineStateChangesConfiguration',
+            iamRoleArn: props.buildAsRoleArn,
+            slackChannelId: slackChannelId,
+            slackWorkspaceId: 'T5J1W20JV',
```
Comment (igngzl) [inline src/PipeLineTemplate/lib/notification.ts:25]: maybe move the value to props

File: src/PipeLineTemplate/lib/notification.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import * as ssm from '@aws-cdk/aws-ssm';
+import { BuildOperationsDetails } from "./buildOperationsDetails";
+import { IPipeline } from "@aws-cdk/aws-codepipeline";
+
+export class Notification extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: BuildOperationsDetails, pipeline: IPipeline) {
+        super(scope, id);
+        
+        let slackChannelName = 'sphinx-env-';
+        if(props.githubRepositoryBranch === 'master' || props.githubRepositoryBranch === 'bumastemra-uat'){
```
Comment (Grendel) [inline src/PipeLineTemplate/lib/notification.ts:12]: Can we change the second part to `props.githubRepositoryBranch.startsWith('bumastemra')`

That will cover `bumastemra` (PROD) also.

File: src/PipeLineTemplate/lib/notification.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as ssm from '@aws-cdk/aws-ssm';
+import { BuildOperationsDetails } from "./buildOperationsDetails";
+import { IPipeline } from "@aws-cdk/aws-codepipeline";
+
+export class Notification extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: BuildOperationsDetails, pipeline: IPipeline) {
+        super(scope, id);
+        
+        let slackChannelName = 'sphinx-env-';
```
Comment (AbdoNile) [inline src/PipeLineTemplate/lib/notification.ts:11]: can you read all constants from a parameter , which populates from ENV Variable at the entry point ?

File: src/PipeLineTemplate/lib/notification.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+        let slackChannelId = ssm.StringParameter.valueForSecureStringParameter(scope, slackChannelName, 1);
+
+        new chatbot.CfnSlackChannelConfiguration(this, props.projectName + 'SlackChannelConfiguration', {
+            loggingLevel: 'INFO',
+            configurationName: props.projectName + 'CodePipelineStateChangesConfiguration',
+            iamRoleArn: props.buildAsRoleArn,
+            slackChannelId: slackChannelId,
+            slackWorkspaceId: 'T5J1W20JV',
+            // snsTopicArns: [notificationTopic.topicArn],
```
Comment (AbdoNile) [inline src/PipeLineTemplate/lib/notification.ts:26]: please remvoe commented code

**Pull request merged**